MIGRATION_GUIDE.md

```diff
@@ -14,6 +14,7 @@ Looking to upgrade from Sentry SDK 2.x to 3.x? Here's a comprehensive list of wh
 - You can no longer change the sampled status of a span with `span.sampled = False` after starting it.
 - The `Span()` constructor does not accept a `hub` parameter anymore.
 - `Span.finish()` does not accept a `hub` parameter anymore.
+- `Span.finish()` no longer returns the `event_id` if the event is sent to sentry.
```

Comment:
isnt this a problem? I remember that we need the event_id returned by capture_event() or else the user feedback modal can not be used?

Reply:
user feedback is for error events, not transactions